File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/views/login/EnterprisePortalView.java
package app.editors.manager.mvp.views.login;

import moxy.viewstate.strategy.OneExecutionStateStrategy;
import moxy.viewstate.strategy.StateStrategyType;

import app.editors.manager.mvp.views.base.BaseView;

@StateStrategyType(OneExecutionStateStrategy.class)
public interface EnterprisePortalView extends BaseView {
    void onSuccessPortal(String portal);
    void onHttpPortal(String portal);
    void onPortalSyntax(String message);
    void onLoginPortal(String portal);
    void onShowDialog();
}

File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/views/login/EnterprisePortalView.kt
package app.editors.manager.mvp.views.login

import app.editors.manager.mvp.views.base.BaseView
import moxy.viewstate.strategy.OneExecutionStateStrategy
import moxy.viewstate.strategy.StateStrategyType

@StateStrategyType(OneExecutionStateStrategy::class)
interface EnterprisePortalView : BaseView {
    fun onSuccessPortal(portal: String, providers: Array<String>)
    fun onHttpPortal(portal: String, providers: Array<String>)
    fun onPortalSyntax(message: String)
    fun onLoginPortal(portal: String)
    fun onShowDialog()
}